public/i18n/en/translation.json

```diff
@@ -1095,7 +1110,49 @@
       "local_database": "Local database",
       "custom_option_combobox_placeholder": "Search key",
       "strong": "Strong",
-      "weak": "Weak"
+      "weak": "Weak",
+      "roadwarrior_server_configured": "Road Warrior server configured",
+      "username": "Username",
+      "expiration": "Expiration",
+      "reserved_ip": "Reserved IP",
+      "connection": "Connection",
+      "connected": "Connected",
+      "not_connected": "Not connected",
+      "download_configuration": "Download configuration",
+      "download_certificate": "Download certificate",
+      "download_qr_code": "Download QR code",
+      "regenerate_certificate": "Regenerate certificate",
+      "add_vpn_account": "Add VPN account",
+      "password_not_configured": "Password not configured",
+      "edit_user_database": "Edit user database",
+      "no_users_found": "No users found",
+      "connection_all": "Connection: All",
+      "connection_connected": "Connection: Connected",
+      "connection_not_connected": "Connection: Not connected",
+      "delete_account": "Delete account",
+      "delete_account_message": "This action will also delete the certificate, if present.",
+      "user": "User",
+      "certificate_expiration_days": "Certificate expiration (days)",
+      "edit_account": "Edit account",
+      "edit_vpn_account": "Edit VPN account",
+      "add_account": "Add account",
+      "choose_user_from_database": "Choose user from database",
+      "renew_certificate": "Renew certificate",
+      "renew": "Renew",
+      "renew_certificate_confirm_message": "Renew certificate for '{username}'? This will revoke the current certificate, download the new certificate to reconnect.",
```

Comment:
Little improvement.
```suggestion
      "renew_certificate_confirm_message": "Renew certificate for '{username}'? This will revoke the current certificate. You will need to download the new certificate to reconnect.",
```